File: ObjOriPro/Lab4/Persoana.java
public class Persoana
{
	private String nume;
	private String prenume;
	private String cnp;
	private int anulnasterii;
	public Persoana(String nume,String prenume,String cnp,int anulnasterii)
	{
		this.nume=nume;
		this.prenume=prenume;
		this.cnp=cnp;
		this.anulnasterii=anulnasterii;
	}
	public String getNume()
	{
		return nume;
	}
	public String getPreume()
	{
		return prenume;
	}
	public String getCnp()
	{
		return cnp;
	}
	public int getAnulnasterii()
	{
		return anulnasterii;
	}
}

File: TAP/Lab6/Lab6pb1/Persoana.cpp
#include<iostream>
#include"Persoana.h"
using namespace std;

Persoana::Persoana(string CNP, string nume) :CNP(CNP), nume(nume){}
Persoana::Persoana(Persoana& other)
{
	this->CNP = other.CNP;
	this->nume = other.nume;
}